src/jaxsim/api/contact.py

```diff
@@ -7,6 +7,7 @@
 import jaxsim.rbda
 import jaxsim.typing as jtp
 
+from . import link as Link
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/119/commits/dd0d00b4c76315b48a2a096157ff6531e5e14037